### PR TITLE
IMA-10467 - Add converter from txt to JSON for appthinningreport

### PR DESCRIPTION
**Pull request by vshaweddy-cpi**
**Change Description:**
Convert app thinning report.txt to json file
**Test Plan/Testing Performed:**
Run using xcode

### CODE REVIEW

**Comment by vshaweddy-cpi** [comment]
Scheme settings
Screen Shot 2021-04-06 at 12 03 22 PM

**Review by mgorkani-cpi — CHANGES_REQUESTED** [review]

File: Sources/Converter/Helper/MemorySize.swift [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+}
+
+extension MemorySize {
+    var displayString: String {
+        switch bytes {
+        case 0..<1_024:
+            return "\(bytes) bytes"
+        case 1_024..<(1_024 * 1_024):
+            return "\(String(format: "%.2f", kilobytes)) kb"
+        case 1_024..<(1_024 * 1_024 * 1_024):
```
[the remark this answers is not in the document]
Comment (vshaweddy-cpi) [reply Sources/Converter/Helper/MemorySize.swift]: The first condition 1_024..<(1_024 * 1_024) will force the third condition to be (1_024*1_024)..<(1_024 * 1_024 * 1_024). But I also update it to be more concise.

**Comment by masumh** [comment]
{"export_url":"https://github.com/settings/security-log/export.json?token=6127babc-b98d-4664-8d00-95581e5195c1","job_url":"https://github.com/_jobs/6127babc-b98d-4664-8d00-95581e5195c1"}

File: Package.swift [hunk 2]
```diff
@@ -25,10 +29,13 @@ let package = Package(
...
         .target(
             name: "xcparse",
-            dependencies: [ "XCParseCore", "SPMUtility" ]),
+            dependencies: [ "XCParseCore", "SPMUtility", "Converter" ]),
         .target(
             name: "XCParseCore",
             dependencies: [ "SPMUtility" ]),
+        .target(
+            name: "Converter",
+            dependencies: []),
```
Comment (masumh) [inline Package.swift:38]: .target(
            name: "Converter",
            dependencies: []),

**Review by masumh — APPROVED** [review]

**Comment by masumh** [comment]
https://github.com/ChargePoint/xcparse/pull/57#issue-609242851

**Review by masumh — APPROVED** [review]
https://github.com/ChargePoint/xcparse/pull/57#issue-609242851

**Review by masumh — APPROVED** [review]
https://github.com/ChargePoint/xcparse/pull/57#issue-609242851

**Comment by masumh** [comment]
https://github.com/ChargePoint/xcparse/pull/57#issue-609242851

**Comment by rsukumar-cpi** [comment]
Picked up these changes and created a new PR #73.

**Pull request closed without merging**